core-header-panel.html

```diff
@@ -146,11 +146,11 @@
 
       <div id="mainPanel" class="layout vertical flex">
 
-        <div id="mainContainer" class-name="[[_mainContainerClass]]">
+        <div id="mainContainer" class$="[[_computeMainContainerClass(mode)]]">
           <content id="mainContent" select="*"></content>
         </div>
 
-        <div id="dropShadow" class-name="[[_dropShadowClass]]"></div>
+        <div id="dropShadow" hidden$="[[_computeDropShadowHidden(_atTop, mode, shadow)]]"></div>
```

Comment:
Hm unsure, but I don't think so. This component wants only a bottom drop shadow: https://www.polymer-project.org/0.5/components/core-header-panel/demo.html

Also, this is a `core-` thing, and importing a `paper-` thing seems Bad:registered:?